watertap/core/zero_order_costing.py

```diff
@@ -2969,6 +2969,14 @@ def cost_microbial_battery(blk):
         blk.config.flowsheet_costing_block.cost_flow(
             blk.unit_model.electricity[t0], "electricity"
         )
+        blk.config.flowsheet_costing_block.cost_flow(
+            blk.unit_model.properties_in[t0].flow_mass_comp["filtration_media"],
+            "filtration_media",
+        )
+        blk.config.flowsheet_costing_block.cost_flow(
+            blk.unit_model.properties_byproduct[t0].flow_mass_comp["filtration_media"],
+            "filtration_media_disposal",
```

Comment:
Should this be at the flowsheet level instead?

Reply:
Are you referring to just the filtration media disposal cost? I don't think it hurts anything to have it here, but I'm open to considering an argument to the contrary. You mentioned to me the other day that you thought it made sense to include spent media disposal costs at the flowsheet level, since for this process it is conceivable that we might want to model something different downstream for the spent media other than just simply disposing of it. But I figure in that case, we could just set filtration_media_disposal_cost to $0/kg, and then add whatever downstream model units we wanted.

Reply:
> But I figure in that case, we could just set filtration_media_disposal_cost to $0/kg, and then add whatever downstream model units we wanted.

That works for me.